staubli_val3_driver/val3/ros_server/dataReceiver.pgx

```diff
@@ -75,4 +75,4 @@
   endWhile
 end]]></Code>
   </Program>
-</Programs>
+</Programs>
```

Comment:
Not sure this is really an issue, but all files appear to be losing their newline at the end of the file.

Reply:
My guess is that the original application was written with an older version of SRS (2013 or 2016) and it was an automatic change. It shouldn't cause an issue with backwards compatibility (loads fine on CS8)